### DISCUSSION TITLE
兄弟姊妹

### DISCUSSION THREAD

**Discussion by lai1008** [1101-GS4719-C/Python-Community-GS4719-C#2281] [General]
不知道大家在家是否有兄弟姊妹，很好奇大家跟自己兄弟姊妹的相處模式，會是和諧的還是比較火爆的或是還有其他可能?

**Comment by Mminniehuang** [comment]
我有一個小我15歲的弟弟，可能是我們的年紀相差很大，所以我們的相處模式大概是每天被他吵到極點，每天一起床就是陪我玩陪我玩，還要把屎把尿照顧他，我感覺我的身分有點像老母親的樣子，雖然還是會常常被他的童言童語笑到，例如我要坐後面結果默默去開前面的門，就是一個前後分不清的弟弟。

**Comment by christaaah** [comment]
我有一個弟弟，年齡只差我一歲，之前還在家的時候我們幾乎天天都會吵起來，有時候一開始很和諧但最後還是會吵起來，但現在我們兩個都不在家，反而變得真的很和諧，偶爾見面也是相處很開心。但如果在家時間久了有會恢復以前狀態ww

**Comment by Elaine4488** [comment]
我有一個姐姐跟一個妹妹，年齡都很接近，感覺相處起來就像朋友一樣。吵架的時候我常常被迫扮演調停者的角色，雖然這角色有時候有點累，但也讓我學會怎麼安撫雙方的情緒，現在想想還挺實用的。而且我們吵歸吵，事後總是很快就和好，可能是因為大家都很直接，吵完反而更了解彼此的想法。有時候也會覺得有兄弟姊妹其實是一件很幸運的事，不管是聊八卦、分享生活，還是需要有人支持的時候，他們都會是最了解我的人。

**Comment by yue083139** [comment]
我有一個大兩歲的姐姐，平常在家屬於那種很強勢的人，對於很多事情都有自己的主見和看法，我個人覺得我們之間的相處還挺愉快的，畢竟我是那種逆來順受比較沒啥意見的人，有時候有人會幫我做決定也是蠻爽的，最近姐姐也出國讀書沒啥機會見到面，感覺還有點感傷的感覺，只能每天傳訊息聊天而已。

**Comment by qiu0109** [comment]
我有哥哥跟弟弟，小時候的相處模式大概是我跟哥哥一起欺負弟弟，或是大家互相看不順眼，但長大以後感情就還不錯，但也可能是因為三個人都在不同縣市的關係，沒有常常住在一起就不會有爭吵。

**Comment by Tchang323** [comment]
我沒有兄弟姊妹，但一直都很希望可以有個哥哥XD，但聽到很多有兄弟姊妹的朋友，他們都很羨慕我是獨生女哈哈

**Comment by lichilun** [comment]
我是獨生女，但我超希望有一個姊姊或妹妹，看到朋友有姐妹超羨慕，感覺可以有多一個人說心事，我喜歡很多人聚在一起的感覺，再家一個人時常感到孤獨。
